@xen-orchestra/web/src/components/site/dashboard/Backups.vue

```diff
@@ -53,4 +56,28 @@ const segments = computed<DonutChartWithLegendProps['segments']>(() => [
     color: 'disabled',
   },
 ])
+
+const vmsProtectionTitle = computed<DonutChartWithLegendProps['title']>(() => ({
+  label: t('backups.vms-protection'),
+  iconTooltip: t('backups.vms-protection.tooltip'),
+  icon: faCircleInfo,
+}))
+
+const vmsProtectionSegments = computed<DonutChartWithLegendProps['segments']>(() => [
+  {
+    label: t('backups.vms-protection.protected'),
+    value: record.value?.backups?.vmsProtection?.protected ?? 0,
```

Comment:
If `backups` is defined, `vmsProtection` will be present.

```suggestion
    value: record.value?.backups?.vmsProtection.protected ?? 0,
```